plugins/module_utils/constants.py

```diff
@@ -157,3 +157,7 @@
     "delete",
     "update",
 )
+
+INTERFACE_FLOW_RULES_TYPES_MAPPING = {"port_channel": "PORTCHANNEL", "physical": "PHYSICAL", "l3out_sub_interface": "L3_SUBIF", "l3out_svi": "SVI"}
+
+INTERFACE_FLOW_RULES_STATUS_MAPPING = {"enabled": "ENABLED", "disabled": "DISABLED"}
```

Comment:
do we need mapping here or can we just capitalise in code?

Reply:
to be consistent with the way we do things with our Ansible modules, I kept it in small caps.